encoding/json/sonic.go

```diff
@@ -0,0 +1,20 @@
+//go:build sonic && !(darwin || arm64)
```

Comment:
I'm not seeing anything that suggests mac or arm64 is incompatible with this. They may have added more support but I was questioning this at the time you raised the PR too.

Reply:
[fixed](https://github.com/thrasher-corp/gocryptotrader/pull/1623/commits/1697977b1ddaba0ea9a78cee01c156fcf607e7ec)